lib/client/identityfile/identity.go

```diff
@@ -630,9 +630,10 @@ func NewClientStoreFromIdentityFile(identityFile, proxyAddr, clusterName string)
 
 	// Save temporary profile into the key store.
 	profile := &profile.Profile{
-		WebProxyAddr: proxyAddr,
-		SiteName:     key.ClusterName,
-		Username:     key.Username,
+		WebProxyAddr:     proxyAddr,
+		SiteName:         key.ClusterName,
+		Username:         key.Username,
+		PrivateKeyPolicy: keys.GetPrivateKeyPolicy(key.PrivateKey),
```

Comment:
Breakage seems legit, are we missing previous backports?

Reply:
Yes, https://github.com/gravitational/teleport/pull/20946 needs to merge first